ogc/features/url.go

```diff
@@ -153,3 +175,90 @@ func clone(params url.Values) url.Values {
 	}
 	return copyParams
 }
+
+func parseLimit(params url.Values, limitCfg engine.Limit) (int, error) {
+	limit := limitCfg.Default
+	var err error
+	if params.Get(limitParam) != "" {
+		limit, err = strconv.Atoi(params.Get(limitParam))
+		if err != nil {
+			err = fmt.Errorf("limit must be numeric")
+		}
+		// OpenAPI validation already guards against exceeding max limit, this is just a defense in-depth measure.
+		if limit > limitCfg.Max {
+			limit = limitCfg.Max
+		}
+	}
+	if limit < 0 {
+		err = fmt.Errorf("limit can't be negative")
+	}
+	return limit, err
+}
+
+func parseBbox(params url.Values) (*geom.Extent, int, error) {
+	bboxCrs, err := parseSRID(params, bboxCrsParam)
+	if err != nil {
+		return nil, -1, err
+	}
+
+	if params.Get(bboxParam) == "" {
+		return nil, bboxCrs, nil
+	}
+	bboxValues := strings.Split(params.Get(bboxParam), ",")
+	if len(bboxValues) != 4 {
+		return nil, bboxCrs, fmt.Errorf("bbox should contain exactly 4 values " +
+			"separated by commas: minx,miny,maxx,maxy")
+	}
+
+	var extent geom.Extent
+	for i, v := range bboxValues {
+		extent[i], err = strconv.ParseFloat(v, 64)
+		if err != nil {
+			return nil, bboxCrs, fmt.Errorf("failed to parse value %s in bbox, error: %w", v, err)
+		}
+	}
+
+	return &extent, bboxCrs, nil
+}
+
+func parseSRID(params url.Values, paramName string) (int, error) {
+	srid := wgs84SRID
+	param := params.Get(paramName)
+	if param == "" {
+		return srid, nil
+	}
+	param = strings.TrimSpace(param)
+	if !strings.HasPrefix(param, crsURLPrefix) {
+		return srid, fmt.Errorf("%s param should start with %s, got: %s", paramName, crsURLPrefix, param)
+	}
+	lastIndex := strings.LastIndex(param, "/")
+	if lastIndex != -1 {
+		crsCode := param[lastIndex+1:]
+		if crsCode == wgs84CodeOGC {
+			return srid, nil // CRS84 is WGS84, just like EPSG:4326 (only axis order differs but SRID is the same)
```

Comment:
maar dat is toch wel een wezenlijk verschil? als iemand een bbox opgeeft in epsg:4326 en het wordt geinterpreteerd als ogc-wgs84 dan worden de coordinaten verkeerd om gelezen